pkg/cmd/server/origin/handlers.go

```diff
@@ -21,30 +21,32 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// TODO once we have a MuxHelper we will not need to hardcode this list of paths
+// RootPaths are paths we server on from root
+var RootPaths kapi.RootPaths = kapi.RootPaths{Paths: []string{
+	"/api",
+	"/api/v1beta3",
```

Comment:
@stevekuznetsov Can you get reasonable access to the master config as this point to build the correct list of versioned endpoints?  If so, may as well specify them correctly to stay consistent with upstream, but correct to our config.

That would require a second integration test.